Sources/Persistence/PersistencePerformanceMetricsTracker.swift

```diff
@@ -105,7 +105,7 @@ public extension PersistencePerformanceMetricsTracker {
             try execute { result in
 
                 switch result {
-                case .success(let blobSize, let memSize):
+                case .success((let blobSize, let memSize)):
```

Comment:
I think we can omit these for now because the CI hasn't been update do Xcode 11.4 yet, so this won't break our CI.

It's best to leave this to a proper Xcode 11.4 / Swift 5.2 migration 😉

Reply:
Makes sense, I'll leave it as it was.